LiveCalcCore/CalculatorViewModel.cs

```diff
@@ -0,0 +1,167 @@
+﻿using CommunityToolkit.Mvvm.ComponentModel;
+using CommunityToolkit.Mvvm.Input;
+
+namespace CalcCore;
+
+public partial class CalculatorViewModel : ObservableObject
+{
+    [NotifyPropertyChangedFor(nameof(Formula))]
+    [ObservableProperty]
+    private CalculatorCommand _pushedCommand = CalculatorCommand.None;
+
+    [ObservableProperty]
+    private Value _displayValue = new();
+
+    [ObservableProperty]
+    private Value _memoryValue = new();
+
+    [ObservableProperty]
```

Comment:
This pushed value also needs `[NotifyPropertyChangedFor(nameof(Formula))]`

And maybe it's obvious, but the Formula property is for adding this part of the calculator:

<img width="668" height="404" alt="image" src="https://github.com/user-attachments/assets/151ccf80-a945-4caf-badf-53184a277ef2" />